phantomas/mr_simul/synthetic.py

```diff
@@ -29,20 +29,18 @@ class AxiallySymmetricModel():
 
     """
 
-    def signal(self, qnorms, thetas, tau=1 / (4 * np.pi**2)):
+    def signal(self, bvals, thetas):
         r"""
         Returns the simulated signal attenuation. The angles thetas correspond
         to the angles between the sampling directions and the principal axis
         of the diffusion model. Must be implemented in subclasses.
 
         Parameters
         ----------
-        qnorms : array-like shape (K, )
-            The norm of q vectors in mm\ :superscript:`-1`.
+	bvals : array-like shape (K, )
+	    B-values. 
```

Comment:
Sounds great! Could you also add the dimension of b-values (should be ``[s\ mm\ :superscript:`-2`]`` I think)